app/Keycloak/Events/ClientDisabled.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Events;
+
+use Illuminate\Foundation\Events\Dispatchable;
+use Ramsey\Uuid\UuidInterface;
+
+final class ClientDisabled
```

Comment:
Maybe we reuse the terminology from `Auth0` being `ClientBlocked` and `UiTiD` being `ConsumerBlocked` instead of using Enable/Disable because we are inside the publiq-platform domain space.